db/migrate/20170921115315_create_users.rb

```diff
@@ -1,16 +1,13 @@
 class CreateUsers < ActiveRecord::Migration[5.1]
   def change
     create_table :users do |t|
-      t.belongs_to :level, foreign_key: true
-      t.belongs_to :role, foreign_key: true
-
+      t.belongs_to :role, index: true
```

Comment:
It is not a good approach to edit existing migration, just FYI

Reply:
Yes, but it is wrong migration, which doesn't work when run db:migration and crush all other migration. it's my mistake from first PR

Reply:
How did you run it the first time then?

Reply:
with commented relationship, and before commit I delete comment :(

Reply:
I already said that it is bad practice to write the code in advance! Migration with comments -- I do not know what to say here...

Reply:
Situation is catch 22 and Yes it is bad but also we need this so any good suggestion on this one?